packages/fxa-settings/src/components/ConfirmWithLink/index.stories.tsx

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import ConfirmWithLink from '.';
+import AppLayout from '../../components/AppLayout';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+import { Subject } from './mocks';
+
+export default {
+  title: 'components/ConfirmWithLink',
+  component: ConfirmWithLink,
+} as Meta;
+
+const storyWithProps = ({ ...props }) => {
```

Comment:
Q (possibly for @LZoog) but should we be doing something similar to the fanciful `React.ComponentProps` trick from #14815?

```jsx
const storyWithProps = (props: React.ComponentProps<typeof CardHeader>) => {
```

Or maybe not, that might be the only instance of ComponentProps currently in the repo, per a quick olde <kbd>git grep "React.ComponentProps" packages</kbd>.

Reply:
`React.ComponentProps<typeof CardHeader>` is great in some circumstances like...

```
const storyWithProps = (
  props?: React.ComponentProps<typeof Subject>,
  storyName?: string,
) => {
  const story = () => <Subject {...props} />;
  if (storyName) story.storyName = storyName;
  return story;
};
```

... if you have a `Subject` defined that doesn't take the exact same props as the component. Otherwise, it does save an export/import, but I think importing the type is fine.